src/components/shared/Entity/Details/Logs/Status/Overview/DetailWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Typography } from '@mui/material';
+import { diminishedTextColor } from 'context/Theme';
+import { useIntl } from 'react-intl';
+import { DetailWrapperProps } from './types';
+
+export default function DetailWrapper({
+    children,
+    headerMessageId,
+    Hydrating,
+}: DetailWrapperProps) {
+    const intl = useIntl();
+
+    return (
+        <Box>
+            <Typography
+                noWrap
+                sx={{
+                    color: (theme) => diminishedTextColor[theme.palette.mode],
+                    mb: '2px',
+                }}
+            >
+                {intl.formatMessage({ id: headerMessageId })}
+            </Typography>
+
+            {Hydrating ?? children}
```

Comment:
Whenever there is a capitalized props like `Hydrating` I assume it takes in a functional component and used like 

`<Hydrating {...whatever props you might need}/>`

In this scenario it looks like all 3 uses of this are passing in a skeleton with different sizes. I think we should just pass in the sizing they want and then this component itself can handle checking hydration.